templates/indicators/result_form_common_js.html

```diff
@@ -378,20 +378,15 @@
             let displayEvidenceWarning = (evidenceURL && URLValidationResult.valid == 'warn');
             const successHandler = (response, status, jqXHR) => {
                 // save successful! initial message from back-end to include translated indicator level/number:
-                let message = response.success_message;
                 if (displayEvidenceWarning) {
                     // hinky URL means save, but add warning and don't auto-dismiss the success noti:
-                    message += "  {% trans 'However, there may be a problem with the evidence URL.' %}";
-                    message += ' <a href="#" id="warning_review">{% trans "Review warning." %}</a>';
-                    noticeType = 'notice';
-                    hide = false;
-                } else {
-                    // URL is fine, message doesn't need additions and should auto-dismiss
-                    noticeType = 'success';
-                    hide = true;
-                }
-                let notice = window.autoDismissingNotification({type: noticeType, message: message, hide: hide, textTrusted: true});
-                if (displayEvidenceWarning) {
+                    let message_text = "{% trans 'However, there may be a problem with the evidence URL.' %}";
+                    message_text += ' <a href="#" id="warning_review">{% trans "Review warning." %}</a>';
```

Comment:
Maybe formulate this as a single string and add a translator comment?

Reply:
I kept it separate to try and keep HTML out of the translator strings because we'd had problems with it - maybe with comments it can be clear?